etc/kayobe/ansible/configure-vxlan.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Configure VXLAN
   hosts: controllers,compute,seed,storage
-  gather_facts: false
+  gather_facts: true
```

Comment:
There is potential for some incorrect behaviour here, since facts are gathered as the bootstrap user using the system python interpreter, some facts may be incorrect. In general we disable fact gathering during the user & venv bootstrap phase of host configure to avoid this. I think this is mostly an issue when using fact caching, but I can't remember all the details.

Let's try it like this, but keep an eye out for odd owner/permissions issues.